cov.py

```diff
@@ -76,31 +76,39 @@ def sim_cov(out, cfg, cwd, opts_vec, opts_cov, csv_list):
   test_name = "riscv_instr_cov_test"
   base_sim_cmd = ("python3 %s/run.py --simulator %s --simulator_yaml %s --noclean "
                   "--so -o %s --cov -tl %s %s "
-                  "-tn %s --steps gen --sim_opts \"<trace_csv_opts> %s %s\" " %
+                  "-tn %s --steps gen --sim_opts \"<trace_csv_opts> %s %s <visualization>\" " %
                   (cwd, argv.simulator, argv.simulator_yaml, out, argv.testlist,
                    argv.opts, test_name, opts_vec, opts_cov))
+  if argv.simulator == "pyflow" and argv.enable_visualization: 
```

Comment:
could something like this work instead?
```python
if argv.simulator == "pyflow" and argv.enable_visualization:
  base_sim_cmd += (" --enable_visualization")
```
it might simplify the logic a little bit, wdyt?

Reply:
Actually `--enable_visualization` is not known by the `run.py` (it's an command line argument of `riscv_instr_cov_test`). Hence, it has to come right after `<trace_csv_opts> %s %s` and **inside double quotes**, that's why I've used `re.sub("<visualization>", "--enable_visualization", base_sim_cmd)`. `base_sim_cmd += (" --enable_visualization")` won't work because it just appends `--enable_visualization` to the end of command.

Reply:
yup you're right, I missed the double quotes the first time through. thanks for pointing that out!